dir/path.go

```diff
@@ -65,7 +65,7 @@ func (p *PathManager) Config() string {
 //
 // extension: support .crt|.key
 func (p *PathManager) Localkey(name string, extension string) string {
```

Comment:
I don't think any caller would be interested in just retrieving cert or key path. Can we return both paths instead of one at a time?

Reply:
Good idea! Updated.